docs/components/content/Playground.vue

```diff
@@ -1,52 +1,83 @@
 <script setup>
-const PARSE_SERVER = 'https://mdc.nuxt.dev/api/parse'
+import { parse } from '../../../src/runtime/markdown-parser'
```

Comment:
Including server-related logic in the app, the runtime is a bad idea and anti-pattern, even for docs and playground, and by lazy loading, we are making total bundle size at least 3x! (same parser to server/SSR/Client). 

We could use a local API with the keep-alive connection that reduces delay or even service-worker target of nitro, making it zero latency.

Reply:
I personally don't see an issue since it's loaded on-demand:

https://user-images.githubusercontent.com/904724/169517170-113d5fb6-34f8-4db4-afa2-9c618f27dd90.mp4


Also the API for it is bad in term of UX (we tried with https://mdc.nuxt.dev) and consume unnecessary server resources and cost.

Reply:
You just downloaded 8MB in the above recording + Hidden overhead on the browser to load all of this :) As I mentioned, we could benefit from SW target if you concern about hosting costs...

Reply:
Hey :)

Using Service Worker was our first objective for this playground, but no chance during implementation.

Could you maybe point us on examples or instructions on how to do it?


Would love to get help on these points as mentioned in the commit above.

Reply:
> You just downloaded 8MB in the above recording + Hidden overhead on the browser to load all of this :) As I mentioned, we could benefit from SW target if you concern about hosting costs...

Happy to improve it on another PR but the benefits of the playground in term of documentation is a must have and worth the extra MB (which are cached for future navigation)

Reply:
I would be more than happy to help make the service worker idea work. It is probably even already possible with nitro preset will try on a small demo! But it needs implementation to still be in the server and changes in this PR is against it!

Reply:
Most likely this can be started from another branch (`main` still has server-only `parse`), and merged to this one if this is reliable solution.

I know @farnabaz tried to use SW from Nitro without chance.

Reply:
Sure! As long as we are in the same context that this usage in playground needs to be reverted, I'm fine to not block the changes. Will provide you an example repo. BTW you can always DM me when having issues ;)

Reply:
Still I don't believe SW is the solution here, I would use a Web Worker to load this code and execute it to not block the main thread (even though it is quite fast actually)

Reply:
Would be happy to discuss about benefits of web-worker approach :)